Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -100,7 +100,9 @@ let Keyboard = {
    * - `keyboardDidChangeFrame`
    *
    * Note that if you set `android:windowSoftInputMode` to `adjustResize`  or `adjustNothing`,
-   * only `keyboardDidShow` and `keyboardDidHide` events will available on Android.
+   * only `keyboardDidShow` and `keyboardDidHide` events will be available on Android and
+   * `keyboardWillShow` as well as `keyboardWillHide` are generally not available on Android
+   * since there is no corresponding event.
```

Comment:
`keyboardWillShow` and `keyboardWillHide` are not available on Android. There is just no event for that. (As of now, might of course change in the future (hopefully)).